03-events-forms-refs-life-cycle/events/custom_link/src/CustomLink.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react';
+
+class CustomLink extends Component{
+	render(){
+		var link;
+		if(this.props.url !== "#"){
+
```

Comment:
you can simply have a prop called `disabled` which is based on the parent component's `this.state.disabled`. If the `CustomLink` has `props.disabled` then render `#`, otherwise render a clickable link.